cmd/buildah/logout.go

```diff
@@ -45,6 +45,12 @@ func logoutCmd(c *cobra.Command, args []string, iopts *logoutReply) error {
 	if len(args) == 0 && !iopts.all {
 		return errors.Errorf("registry must be given")
 	}
+
+	_, err := getStore(c)
```

Comment:
Can you do this on one line please.
if _, err:=getstore(); err != nil

Reply:
fixed